reference/configuration/framework.rst

```diff
@@ -142,26 +237,55 @@ is set, then the ``ide`` option will be ignored.
 test
 ~~~~
 
-**type**: ``Boolean``
+**type**: ``boolean``
 
 If this configuration parameter is present (and not ``false``), then the
 services related to testing your application (e.g. ``test.client``) are loaded.
 This setting should be present in your ``test`` environment (usually via
-``app/config/config_test.yml``). For more information, see :doc:`/book/testing`.
+``app/config/config_test.yml``).
 
-.. _reference-framework-trusted-proxies:
+.. seealso::
 
-trusted_proxies
-~~~~~~~~~~~~~~~
+   For more information, see :doc:`/book/testing`.
 
-**type**: ``array``
+default_locale
+~~~~~~~~~~~~~~
 
-Configures the IP addresses that should be trusted as proxies. For more details,
-see :doc:`/cookbook/request/load_balancer_reverse_proxy`.
+**type**: ``string`` **default**: ``'en'``
 
-.. versionadded:: 2.3
-    CIDR notation support was introduced in Symfony 2.3, so you can whitelist whole
-    subnets (e.g. ``10.0.0.0/8``, ``fc00::/7``).
+If the locale can't be determined from the Request, the default locale will be
+used. It becomes the service container parameter called
+``kernel.default_locale``.
+
+.. seealso::
+
+    You can read more information about the default locale in
+    :ref:`book-translation-default-locale`.
+
+trusted_hosts
+~~~~~~~~~~~~~
+
+**type**: ``array`` | ``string`` **default**: ``array()``
+
+A lot of different attacks have been discovered relying on inconsistencies
+between the handling of the ``Host`` header by various software (web servers,
+reverse proxies, web frameworks, etc.). Basically, everytime the framework is
+generating an absolute URL (when sending an email to reset a password for
+instance), the host might have been manipulated by an attacker.
+
+.. seealso::
+
+    You can read "`HTTP Host header attacks`_" for more information about these
+    kinds of attacks.
+
+The Symfony :method:`Request::getHost()
+<Symfony\\Component\\HttpFoundation\\Request:getHost>` method might be
+vulnerable to some of these attacks because it depends on the configuration of
+your web server. One simple solution to avoid these attacks is to whitelist the
+hosts that your Symfony application can respond to. That's the purpose of this
+``trusted_hosts`` option. If the incoming request's hostname doesn't match one
+in this list, the application won't respond and the user will receive a 500
+response.
```

Comment:
But this is good, because we don't cover `trusted_hosts` elsewhere, so we can put all the details we want here.